### PR TITLE
v4 fix arrivejs '!='

### PR DESCRIPTION
**Pull request by youngyou**
Fix default selector for arrive, the '!=' selector works in jquery only, but not arrivejs. 
Fix arrivejs callback.#958

### CODE REVIEW

**Comment by rosskevin** [comment]
Please reference your issue number in the comments so it links them for review. Numbers in titles don't create those links so we don't have the context/test case

File: js/src/bootstrapMaterialDesign.js [hunk 1]
```diff
@@ -140,8 +140,8 @@ const BootstrapMaterialDesign = (($) => {
 
             // add to arrive if present and enabled
             if (document.arrive && this.config.arrive) {
-              $document.arrive(selector, (element) => {  // eslint-disable-line no-loop-func
-                $(element)[jqueryFn](componentConfig)
+              $document.arrive(selector, function(){  // eslint-disable-line no-loop-func
```
Comment (rosskevin) [inline js/src/bootstrapMaterialDesign.js:143]: This change isn't preferred ES2015 syntax;  I am surprised it passed eslint.  `this` is an instance of `BootstrapMaterialDesign` and not the `element` as far as I can tell (at least it is with the arrow function).

Please test it with the current ES2015 syntax and let me know if that works or I am misunderstanding it.  I would prefer the current syntax with the explicit element passed to the callback fn.

Comment (youngyou) [inline js/src/bootstrapMaterialDesign.js:143]: It works well with `function`. I think use the `function` is just ok under ES2015 . `Arrivejs` just binds the element to `this`, so the `arrow function` does not work. I can't find a better solution except overwrite `arrivejs`.

Comment (rosskevin) [inline js/src/bootstrapMaterialDesign.js:143]: Are you saying that the arrow function as I had it fails?

Comment (youngyou) [inline js/src/bootstrapMaterialDesign.js:143]: Yes.
And we can use my fork, which supports arrow function callback.
https://github.com/youngyou/arrive.git

**Comment by rosskevin** [comment]
@youngyou your fork for arrive looks good, I suggest you submit that as a PR over there.

I'll merge this as-is, when your arrive PR is accepted, let's switch this back to an arrow fn.

**Pull request merged**